core/dbt/parser/schema_test_builders.py

```diff
@@ -282,46 +293,88 @@ def extract_test_args(test, name=None) -> Tuple[str, Dict[str, Any]]:
 
     @property
     def enabled(self) -> Optional[bool]:
-        return self.modifiers.get('enabled')
+        return self.config.get('enabled')
 
     @property
     def alias(self) -> Optional[str]:
-        return self.modifiers.get('alias')
+        return self.config.get('alias')
 
     @property
     def severity(self) -> Optional[str]:
-        sev = self.modifiers.get('severity')
+        sev = self.config.get('severity')
         if sev:
             return sev.upper()
         else:
             return None
 
     @property
     def store_failures(self) -> Optional[bool]:
-        return self.modifiers.get('store_failures')
+        return self.config.get('store_failures')
 
     @property
     def where(self) -> Optional[str]:
-        return self.modifiers.get('where')
+        return self.config.get('where')
 
     @property
     def limit(self) -> Optional[int]:
-        return self.modifiers.get('limit')
+        return self.config.get('limit')
 
     @property
     def warn_if(self) -> Optional[str]:
-        return self.modifiers.get('warn_if')
+        return self.config.get('warn_if')
 
     @property
     def error_if(self) -> Optional[str]:
-        return self.modifiers.get('error_if')
+        return self.config.get('error_if')
 
     @property
     def fail_calc(self) -> Optional[str]:
-        return self.modifiers.get('fail_calc')
+        return self.config.get('fail_calc')
+
+    @property
+    def meta(self) -> Optional[dict]:
+        return self.config.get('meta')
+
+    @property
+    def database(self) -> Optional[str]:
+        return self.config.get('database')
+
+    @property
+    def schema(self) -> Optional[str]:
+        return self.config.get('schema')
+
+    def get_static_config(self):
```

Comment:
The way this method body is defined makes me think there's a better way to do this. My guess is that should entail a bigger refactor though. What are your thoughts on this, @gshank?

Reply:
I agree that this is not a good way to do this and it should be refactored. We will probably be doing more config work on tests in the future and that would be a good time to do it.